app/views/cookie_preferences/new.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= content_for :page_title, "Cookies" %>
+
+<div class="govuk-width-container">
+  <h1 class="govuk-heading-xl">Cookies</h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render partial: 'cookie-preferences-form' %>
+
+      <h2 class="govuk-heading-l">How we use cookies</h2>
+
+      <h3 class="govuk-heading-m">Essential cookies</h3>
+
+      <p class="govuk-body">Essential ‘Find postgraduate teacher training’ cookies include:</p>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__header">
+        <tr>
+          <th class="govuk-table__header">Name</th>
+          <th class="govuk-table__header">Purpose</th>
+          <th class="govuk-table__header">Expires</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        <tr>
+          <td class="govuk-table__cell">_find_teacher_training_session</td>
+          <td class="govuk-table__cell">Stores encrypted session data</td>
+          <td class="govuk-table__cell">When you close your browser</td>
+        </tr>
+        <tr>
+          <td class="govuk-table__cell">consented-to-cookies</td>
+          <td class="govuk-table__cell">Saves your cookie consent settings</td>
```

Comment:
```suggestion
          <td class="govuk-table__cell">Saves your cookies consent settings</td>
```

Reply:
🤔  I'm not sure about whether this should be plural or singular.  I'll check with Paul G

Reply:
Having spoken to Paul G, we both agree that this copy is correct because:

> cookie settings can apply to numerous cookies - the key bit is settings rather than cookie